hot/loader-helpers.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-env commonjs */
+const path = require(`path`);
+
+// Retreive elemName for hot injection from path convention
+//
+// elem name patterns look like this
+//    ./src/.../${elemName}/index.jade
+// OR ./src/.../${elemName}.jade
```

Comment:
Interesting. Not a blocker for merging but at some point this could use some more complete explanation here of how this convention works, i.e. that module file name is expected to match HTML element tag name (so things like multiple element classes in a single file just won't work, right?).

Reply:
That's right.

Will add a comment